### ISSUE TITLE
Nodes stopped syncing as block validation failed 

### ISSUE THREAD

**Issue by trietnguyen267** [cennznet/cennznet#186]
This problem occurs to boot nodes so far in our records

The impacted node will stop syncing new block even though connected peers still very good
```shell=
2020-04-03 01:07:17 Verification failed for block 0x6a9e58e98ba8ad617a7bf7a9249ca1d4f2b4163bc420a4471fd070b901c5446a received from peer: QmZjEa7tWqLWyxPDrJ6JZ7jZ8yTWnyHUF6G5cah3Kev49i, "UnknownBlock: Require header: BlockId::Hash(0xcf825a77423a84d48e2aad2ad190d24f68a072535d5aca2b4a0564eae729f0e3)"
2020-04-03 01:07:17 Verification failed for block 0x6a9e58e98ba8ad617a7bf7a9249ca1d4f2b4163bc420a4471fd070b901c5446a received from peer: Qmb3wzZsdRPUiAyH6s1YZX1bZDU5CeVW6Zxoy4a97SSSH8, "UnknownBlock: Require header: BlockId::Hash(0xcf825a77423a84d48e2aad2ad190d24f68a072535d5aca2b4a0564eae729f0e3)"
2020-04-03 01:07:18 Verification failed for block 0x6a9e58e98ba8ad617a7bf7a9249ca1d4f2b4163bc420a4471fd070b901c5446a received from peer: QmSsGEA79dszPxsXAzrGipkWWtyJo29WKtH3zCJdHuVJ7v, "UnknownBlock: Require header: BlockId::Hash(0xcf825a77423a84d48e2aad2ad190d24f68a072535d5aca2b4a0564eae729f0e3)"
```

**Work-around**: restart node

//Todo
Adding more logs from sync and babe modules

**Comment by trietnguyen267** [comment]
Potentially related to us @holygits 
https://github.com/paritytech/substrate/issues/4971

**Comment by trietnguyen267** [comment]
Error is thrown from this PR, I'll checking offchain worker log
https://github.com/paritytech/substrate/pull/4878/files

**Comment by jordy25519** [comment]
Should be solved in #303, will re-open if it occurs again
